cryk-ui/src/pages/Cryptocurrencies.tsx

```diff
@@ -192,14 +315,15 @@ export default function Cryptocurrencies() {
                         dialogQuery={CREATE_CRYPTOCURRENCY}
                         refetchInput={refetchInput}
                     />
-
-                    <Pagination className="pagination"
-                        count={totalNumberOfCryptocurrencies}
-                        color="primary"
-                        size="large"
-                        page={currentPage}
-                        variant="outlined"
-                        onChange={(event, value) => setCurrentPage(value)} />
+                    {data.cryptocurrenciesInfo.totalCount !== 0 ?
+                        <Pagination className="pagination"
+                            count={totalNumberOfPages}
+                            color="primary"
+                            size="large"
+                            page={currentPage}
+                            variant="outlined"
+                            onChange={(event, value) => setCurrentPage(value)} />
+                        : null}
```

Comment:
We don't need `: null`. `{data.cryptocurrenciesInfo.totalCount !== 0 ? <Pagination ... />}` is enough

Reply:
It doesn't work like this